### DISCUSSION TITLE
Ionic Appflow with Angular/Capacitor app

### DISCUSSION THREAD

**Discussion by Sampath-Lokuge** [ionic-team/capacitor#3507] [Q&A] [answered]
Hi,

I need to build an app for all the platforms. i.e. web/ios/android My Client has an Angular template and he needs to use that. I know where I can add a Capacitor to it and build a Native build + Web app.  But Can I use Ionic Appflow with the Angular/Capacitor app? Please give me the right direction for that. i.e. Doc or URL reference

**Comment by imhoffd** [comment, marked as answer]
@Sampath-Lokuge Appflow will auto-detect Capacitor as long as your project is integrated with the Ionic CLI (run `ionic init` if you have to).

**Reply by Sampath-Lokuge** [reply]
Oh...Got that. Thanks!
